contrib/emacs/hurl-mode.el

```diff
@@ -0,0 +1,136 @@
+;;; hurl-mode.el --- Major mode for hurl  -*- lexical-binding: t; -*-
+
+;; Copyright (C) 2022 Azzam S.A
+
+;; Author: Azzam S.A <vcs@azzamsa.com>
```

Comment:
We can add more names here such this:
- https://github.com/magit/magit/blob/99d9b3008adf72a0c2cdb7df70b5ae03ea9271c8/lisp/magit.el#L5

Reply:
Same, here, could you please remove:

`;; Copyright (C) 2022 Azzam S.A`
`;; Author: Azzam S.A <vcs@azzamsa.com>` 

(you contribution wil be of course attributed to you, you will be able to check it with git)